src/components/Compare/Filters.tsx

```diff
@@ -26,57 +25,38 @@ const Filters = (props: {
   setHomepageScope: React.Dispatch<React.SetStateAction<HomepageLocationScope>>;
   homepageSliderValue: HomepageLocationScope;
 }) => {
-  const {
-    sliderValue,
-    homepageScope,
-    setHomepageScope,
-    homepageSliderValue,
-  } = props;
-
-  const GeoFilterLabels = {
-    [GeoScopeFilter.NEARBY]: 'Nearby',
-    [GeoScopeFilter.STATE]: `${props.stateId}`,
-    [GeoScopeFilter.COUNTRY]: 'USA',
-  };
+  const { setHomepageScope } = props;
 
-  // Last value is set to 99 for styling
-  // so that the final mark falls on the slider and not to the right of it
-  const marks = [
-    {
-      value: 0,
-      label: GeoFilterLabels[GeoScopeFilter.NEARBY],
-    },
-    {
-      value: 50,
-      label: GeoFilterLabels[GeoScopeFilter.STATE],
-    },
-    {
-      value: 99,
-      label: GeoFilterLabels[GeoScopeFilter.COUNTRY],
-    },
+  const homepageFilterLabels = [
+    homepageLabelMap[HomepageLocationScope.COUNTY].plural,
+    homepageLabelMap[HomepageLocationScope.MSA].plural,
+    homepageLabelMap[HomepageLocationScope.STATE].plural,
   ];
 
-  const sliderHandleChange = (event: any, value: any) => {
-    if (props.setGeoScope) {
-      props.setGeoScope(sliderNumberToFilterMap[value]);
+  const locationPageFilterLabels = ['Nearby', `${props.stateId}`, 'USA'];
+
+  const handleLocationPageSelectedOption = (
+    event: React.MouseEvent<HTMLElement>,
+    value: string,
+  ) => {
+    if (props.setGeoScope && value != null) {
```

Comment:
this can just be `if (value != null) {...` since `props.setGeoScope` is always defined